include/device/input.hpp

```diff
@@ -21,24 +21,55 @@ class InputModules {
     mbed::AnalogIn volume;
 
     /// 慣性計測ユニット(imu)
-    MPU6050 mpu;
+    std::unique_ptr<MPU6050> mpu;
 
     /// ジョイスティックの値を読む
     auto read_joy() -> std::pair<float, float>;
 
     /// IMUの値を読む
     auto read_gyro() -> std::array<float, 3>;
 
+    // NOLINTBEGIN(bugprone-easily-swappable-parameters)
+    InputModules(
+        std::pair<mbed::AnalogIn, mbed::AnalogIn> joy, mbed::AnalogIn volume,
+        unique_ptr<MPU6050> mpu) :
+        joy(joy), volume(volume), mpu(std::move(mpu)) {}
```

Comment:
こっちの方がスッキリしてる気がする　直さなくてもいいです

```suggestion
    InputModules(
        std::pair<mbed::AnalogIn, mbed::AnalogIn> joy, mbed::AnalogIn volume,
        std::unique_ptr<MPU6050>&& mpu) :
        joy(joy), volume(volume), mpu(mpu) {}
```